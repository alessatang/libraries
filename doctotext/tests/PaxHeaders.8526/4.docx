30 atime=1389151471.399894576
30 ctime=1389151471.399894576
