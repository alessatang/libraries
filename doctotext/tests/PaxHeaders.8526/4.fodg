30 mtime=1384978086.840497471
30 atime=1389151471.422894483
30 ctime=1389151471.422894483
