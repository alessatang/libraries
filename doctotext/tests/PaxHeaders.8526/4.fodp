30 mtime=1384978086.843497472
30 atime=1389151471.574893864
30 ctime=1389151471.574893864
