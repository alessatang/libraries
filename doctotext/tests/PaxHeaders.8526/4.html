29 atime=1389151471.57589386
29 ctime=1389151471.57589386
