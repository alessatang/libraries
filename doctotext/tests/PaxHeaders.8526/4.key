30 mtime=1379552471.328940161
30 atime=1389151472.230891191
30 ctime=1389151472.230891191
