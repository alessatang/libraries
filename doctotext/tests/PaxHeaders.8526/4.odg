30 atime=1389151472.297890918
30 ctime=1389151472.297890918
