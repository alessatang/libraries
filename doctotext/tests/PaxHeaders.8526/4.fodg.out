30 mtime=1384978086.675497416
30 atime=1389151471.422894483
30 ctime=1389151471.422894483
